_redirects

```diff
@@ -10,3 +10,4 @@ build_for: [standard, managed]
 /docs/v2.2/* /docs/v19.1/:splat 301
 /docs/releases/ /docs/releases/index.html 200!
 /docs /docs/stable/ 301
+/docs/* /docs/stable/404.html 404
```

Comment:
@mjibson, is this ok, or should I set up such redirects for the 404 page in each version directory?

Reply:
This is fine with me.